node/node.go

```diff
@@ -491,3 +587,152 @@ func (nn nexNode) EmitEvent(inNamespace string, inEvent json.RawMessage) error {
 	}
 	return nn.nc.Publish(models.EventAPIPrefix+"."+inNamespace, event)
 }
+
+func (nn nexNode) StoreRunRequest(workloadType, inId string, inRequest *actorproto.StartWorkload) error {
```

Comment:
I believe I said this in another file somewhere, but it seems it would be good to have the first argument be a `context.Context` to thread it through to the various external calls.

Reply:
I think the nexNode has a context that I intend on using 🤔